main.go

```diff
@@ -122,6 +123,9 @@ func parseFlags() *config {
 	flag.DurationVar(&cfg.remoteReadTimeout, "read-timeout", 30*time.Second,
 		"The timeout to use when reading samples to the remote storage.",
 	)
+	flag.DurationVar(&cfg.remoteReadIgnore, "read-ignore", 3600*time.Second,
```

Comment:
/read-ignore/read-ignore-delay/
And maybe add an example of correct argument so we know which format is parsed